main.go

```diff
@@ -277,8 +277,18 @@ func main() {
 			cfgDeps = []depEntry{flagDep}
 		} else {
 			log.Println("Starting whole vndr cycle because no package specified")
-			if err := os.RemoveAll(vd); err != nil {
-				log.Fatal(err)
+			if len(cleanWhitelist) > 0 {
+				var deps []*build.Package
```

Comment:
This `deps` is always nil from what I see. Is this what we want?

Reply:
Ah, yes, I can just pass `nil` 👍

Reply:
Updated 👍